<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUU - Remote Path</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icon/guu.png</normaloff>../icon/guu.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>It looks like qBitTorrent is running on a separate computer.</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Please choose the containing folder of the uploaded files on the remote computer.</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>If this is an error, simply click OK without making changes.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="remotePath"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="okBtn">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
